plugins/module_utils/problem.py

```diff
@@ -7,13 +7,22 @@
 
 __metaclass__ = type
 
+import re
+
+NEW = "101"
+ASSESS = "102"
+RCA = "103"
+FIX = "104"
+RESOLVED = "106"
+CLOSED = "107"
```

Comment:
```suggestion
from enum import Enum

class StateMappingEnum(Enum):
        NEW = "101"
        ASSESS = "102"
        RCA = "103"
        FIX = "104"
        RESOLVED = "106"
        CLOSED = "107"
```

Reply:
Enums were introduced in Python 3.4. If we need to keep support for Python 2.7, then we cannot use them. Should I wrap them into a regular class? Another possibility is to refactor once the support for Python 2.7 is dropped.

Reply:
No issue. We can revisit this when we remove Python 2.7 support.